Doc/library/calendar.rst

```diff
@@ -138,31 +138,55 @@ interpreted as prescribed by the ISO 8601 standard.  Year 0 is 1 BC, year -1 is
 
    :class:`TextCalendar` instances have the following methods:
 
+
+   .. method:: formatday(theday, weekday, width)
+
+      Return a :class:`str` representing a single day formatted with the given width.
```

Comment:
We generally say "string" without saying the exact type because it renders a bit better in a text (for instance, we don't markup "list" when we talk about a "list of [...]").